scss/components/_vr-controls.scss

```diff
@@ -0,0 +1,110 @@
+      bottom: auto;
+      right: 0;
+      @include vendor-prefixes(transform, translateY(-50%));
+      &--touched {
+        background: -moz-linear-gradient(right, rgba(0,0,0,0.6) 0%, rgba(255,255,255,0) 100%);
+        background: -webkit-linear-gradient(right, rgba(0,0,0,0.6) 0%,rgba(255,255,255,0) 100%);
+        background: linear-gradient(to left, rgba(0,0,0,0.6) 0%,rgba(255,255,255,0) 100%);
+        filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#00ffffff', endColorstr='#a6000000',GradientType=1 );
+      }
+    }
+    &--up {
+      top: 0;
+      left: 50%;
+      bottom: auto;
+      right: auto;
+      @include vendor-prefixes(transform, translateX(-50%));
+      &--touched {
+        background: -moz-linear-gradient(top, rgba(0,0,0,0.6) 0%, rgba(255,255,255,0) 100%);
+        background: -webkit-linear-gradient(top, rgba(0,0,0,0.6) 0%,rgba(255,255,255,0) 100%);
+        background: linear-gradient(to bottom, rgba(0,0,0,0.6) 0%,rgba(255,255,255,0) 100%);
+        filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#a6000000', endColorstr='#00ffffff',GradientType=0 );
+      }
+    }
+    &--down {
+      top: auto;
+      left: 50%;
+      bottom: 0;
+      right: auto;
+      @include vendor-prefixes(transform, translateX(-50%));
+      &--touched {
+        background: -moz-linear-gradient(bottom, rgba(0,0,0,0.6) 0%, rgba(255,255,255,0) 100%);
+        background: -webkit-linear-gradient(bottom, rgba(0,0,0,0.6) 0%,rgba(255,255,255,0) 100%);
+        background: linear-gradient(to top, rgba(0,0,0,0.6) 0%,rgba(255,255,255,0) 100%);
+        filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#00ffffff', endColorstr='#a6000000',GradientType=0 );
+      }
+    }
+  }
+  &-container {
+    position: absolute;
+    display: block;
+    top: 15px;
+    left: 15px;
+    width: 5%;
+    max-width: 80px;
+    max-height: 80px;
+    min-width: 40px;
+    min-height: 40px;
+    background-image: url("assets/images/vr-controls.svg");
+    background-position: 0 0;
+    background-size: 100% 100%;
+    opacity: 1;
+    @include vendor-prefixes (transition, opacity .3s ease-in-out);
+    &--hidden {
+      opacity: 0;
+    }
+    &:after {
+      content: "";
+      display: block;
+      padding-bottom: 100%;
+    }
+  }
+  &--type {
+    display: block;
+    width: 23px;
+    height: 17px;
+    background-image: url("assets/images/vr-video-type.svg");
+    background-position: 0 0;
+    background-size: 100% 100%;
+    &--stereoOn {
+      -webkit-filter: drop-shadow(-1px 1px 4px rgba(255,255,255,.7));
+      filter: drop-shadow(-1px 1px 4px rgba(255,255,255,.7));
+    }
+    &--stereoOff {}
+  }
+}
+.oo-video-type {
+  @include vendor-prefixes (transition, all .3s ease-in-out);
+  &:hover,
+  &:active,
+  &:focus {
+    .oo-vr-icon--type {
+      -webkit-filter: drop-shadow(-1px 1px 4px rgba(255,255,255,.7));
+      filter: drop-shadow(-1px 1px 4px rgba(255,255,255,.7));
+    }
+  }
+}
```

Comment:
Do we need responsive styles for smaller player sizes?

Reply:
This part of the code is removed from this userstory and will be changed in newer userstory with using fonts instead of images.